app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -229,11 +263,32 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
               {selectedLeague?.leagueName as string}
             </LinkCustom>
           </nav>
-          <section className="w-full pt-8" data-testid="weekly-picks">
+          <section className="flex flex-col items-center w-full pt-8" data-testid="weekly-picks">
             <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
               Week {week} pick
             </h1>
 
+            {pickHistory.length > 0 && (
+              <section className="flex flex-wrap w-[90%] gap-4 overflow-x-scroll justify-center pb-10 items-center">
+                {pickHistory?.map((logoURL, index) => (
+                  <div
+                    key={logoURL}
+                    className="flex flex-col items-center justify-center border border-border p-2 rounded-lg gap-1"
+                  >
+                    <span className="text-sm">WEEK {index + 1}</span>
+                    <Image
+                      className="league-entry-logo -mt-1.5"
```

Comment:
I really think we ought to stay away from negative margins. You added this to pull the image up closer to the "WEEK" text?

Reply:
@ryandotfurrer yup.  I can remove it if y'all want ... just thought it looked a bit better this way.